great-idea/index.html

```diff
@@ -12,12 +12,164 @@
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
-</head>
 
-  <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
       
+  </head>
+
+  <body>
+    <div class="container">
+      <header>
+        <nav>
+          <a href="services.html">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </nav>
+
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        
+      </header>
+
+
+
+
+      
+
+      <section class="intro-container">
+        <div class="intro">
+          <h1>Innovation<br/>
+          On<br/>
+          Demand </h1>
+          <button class="started">Get Started</button>
+        </div>
+  
+        <img class="introimg" src="img/header-img.png" alt="Image of a code snippet.">
+      </section>
+
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
     
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+      <section class="top-content">
+      <div class="features">
+        <h2>Features</h2>
+          <p>
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+          </p>
+        </div>
+
+      <div class="about">
+        <h2>About</h2>
+          <p>
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+          </p>
+      </div>
+      </section>
+
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    
+    <section class="bottom-content">
+      <div class="services">
+        <h2>Services</h2>
+          <p>
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
```

Comment:
2 top paragraphs and the 3 below them are too close to each other. Try adding margin/padding.